tests/operator/remember.ts

```diff
@@ -131,4 +131,27 @@ describe('Stream.prototype.remember', () => {
     });
     done();
   });
+
+  it ('should work properly with `xs.create()`', (done) => {
```

Comment:
Can you split this into 2 tests? One test that has `shamefullySendNext('foo')` before `remembered.addListener` and another that has every shamefullySendNext after the addListener?